Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -452,26 +471,25 @@ extension ActivityStreamPanel {
             self.presentViewController(controller, animated: true, completion: nil)
         })
 
-        let removeTopSiteAction = ActionOverlayTableViewAction(title: Strings.RemoveFromASContextMenuTitle, iconString: "action_close", handler: { action in
+        let removeTopSiteAction = ActionOverlayTableViewAction(title: Strings.RemoveContextMenuTitle, iconString: "action_remove", handler: { action in
             ASOnyxPing.reportDeleteItemEvent(eventInfo)
             self.hideURLFromTopSites(site.tileURL)
         })
-        
-        let dismissHighlightAction = ActionOverlayTableViewAction(title: Strings.RemoveFromASContextMenuTitle, iconString: "action_close", handler: { action in
+
+        let dismissHighlightAction = ActionOverlayTableViewAction(title: Strings.RemoveContextMenuTitle, iconString: "action_remove", handler: { action in
```

Comment:
I think it's "action_close" since the image isn't showing up in the context menu for the Remove action. Maybe you can change the asset file for that or change the strings pointing to the file!